src/components/Card/Card.jsx

```diff
@@ -199,6 +205,15 @@ const Card = props => {
     return childSize;
   };
 
+  const elmRef = createRef();
+  const [showTooltip, setshowTooltip] = useState(false);
+  const handleTooltip = IsShow => {
```

Comment:
This functionality should be within a useEffect hook, instead of imperatively called on mouseEnter/Leave.